front/app/containers/MainHeader/index.tsx

```diff
@@ -2,42 +2,22 @@
 import React, { useRef } from 'react';
 import { includes } from 'lodash-es';
```

Comment:
This component originally had both the mobile + desktop nav items. I've separated it into two new components instead as that seemed easier to work with and made the file structure more clear.